daml-assistant/daml-helper/src/DA/Daml/Helper/Main.hs

```diff
@@ -122,6 +122,22 @@ commandParser = subparser $ fold
         <*> (ScriptOptions <$> many (strOption (long "script-option" <> metavar "SCRIPT_OPTION" <> help "Pass option to DAML script interpreter")))
         <*> stdinCloseOpt
 
+    navigatorFlag =
+        -- We do not use flagYesNoAuto here since that doesn’t allow us to differentiate
+        -- if the flag was passed explicitly or not.
+        StartNavigator <$>
+        option reader (long "start-navigator" <> help helpText <> completeWith ["true", "false"] <> idm)
+        where
+            reader = eitherReader $ \case
+                -- We allow for both yes and true since we want a boolean in daml.yaml
+                "true" -> Right True
+                "yes" -> Right True
+                "false" -> Right False
+                "no" -> Right False
+                "auto" -> Right True
+                s -> Left ("Expected \"yes\", \"true\", \"no\", \"false\" or \"auto\" but got " <> show s)
+            -- To make things less confusing, we do not mention yes, no and auto here.
```

Comment:
We're using `--start-navigator=yes` in the example in the docs below. Should we change that to `--start-navigator=true` then?

Reply:
Good catch, I’ve switched to `true`.